README.md

```diff
@@ -35,37 +35,40 @@ chmod +x setup.sh
 Installation of Requirements
 
 ```bash
-sudo apt install apache2 mysql-server python python3-pip python-mysqldb libldap2-dev libmysqlclient-dev
+sudo apt install apache2 mysql-server python3 python3-pip python3-mysqldb libldap2-dev libmysqlclient-dev
 sudo python3 -m pip install -r requirements.txt
 ```
 
-Steps for installation of OutcomeBasedCourse
-1. Clone the repository [OutcomeBasedCourse](https://github.com/GreatDevelopers/OutcomeBasedCourse)
-```bash
-git clone https://github.com/GreatDevelopers/OutcomeBasedCourse
-```
-1. Create a database for OutcomeBasedCourse
-```bash
-mysql -u root -p -e "create database outcomebasedcourse;"
-```
+Steps for installation of OutcomeBasedCourse:
+
+1. Clone the repository [OutcomeBasedCourse](https://github.com/      GreatDevelopers/OutcomeBasedCourse).
+
+        git clone https://github.com/GreatDevelopers/OutcomeBasedCourse
+
+1. Create a database for OutcomeBasedCourse.
+
+         mysql -u root -p -e "create database outcomebasedcourse;"
+
 1. Edit settings.py file in OutcomeBasedCourse/ directory. Things to be edited are:
-DATABASES = {
-    "default": {
+   
+        DATABASES = {
+        "default": {
```

Comment:
It will be better if block will be indented like:
```
DATABASES = {
    "default": { 
        "ENGINE": "django.db.backends.mysql",
        "NAME": "Your database name"
        "HOST": "Your MySQl server host",
        "PORT": "Your MySQl server port",
        "USER": "Your MySQL username",
        "PASSWORD": "Your MySQl password",
    } 
}
```